### ISSUE TITLE
Icon for Eufloria HD (Steam)

### ISSUE THREAD

**Issue by hlechner** [numixproject/numix-icon-theme-circle#1912]
Game: **Eufloria HD**
Game ID: [221180](https://steamdb.info/app/221180/)
Description: **Space exploration, conquest, plant growth and bio mechanical evolution. Conquer asteroids in deep space and use their resources to grow and nurture semi-organic plants and creatures to do your bidding. Plan your strategy against AI opponents that all vie for the same resources and offer fierce opposition.**

Note: Hardcoded (uses **Icon=steam**)

Icon: **No icon provided for .desktop file**
Related Images:
![image](https://cloud.githubusercontent.com/assets/11244067/7061522/80abc04e-de69-11e4-99c5-37e8fbd3525d.png)
![image](https://cloud.githubusercontent.com/assets/11244067/7061528/94fd6d72-de69-11e4-90b7-144d8fdfcb60.png)
![image](https://cloud.githubusercontent.com/assets/11244067/7061538/c131d4dc-de69-11e4-9236-2523357473b7.png)

Eufloria HD.desktop:
```
[Desktop Entry]
Name=Eufloria HD
Comment=Play this game on Steam
Exec=steam steam://rungameid/221180
Icon=steam
Terminal=false
Type=Application
Categories=Game;
```

**Comment by Anaeijon** [comment]
Dear Henrique Lechner,

The email sent to david@d-doering.tk with subject
"[numix-icon-theme-circle] Icon for Eufloria HD (Steam) (#1912)"
will be delivered as soon as possible.

Your email address is automatically added to the
recipients 'whitelist'. This means for further
emails to david@d-doering.tk, you will not receive a delivery
notification again.

TK MAILIAS
*************************************************
Looking for a way to Protect Your Identity and 
for a great no-more-spam solution for your email
address reply@reply.github.com? Sign up for TK Mailias.

It's free! Please visit http://www.mail.tk/

**Comment by Foggalong** [comment]
Fixed in 8cf4eabb9e1c308bbc63f6e6e74175f0570ca997
